.github/pull_request_template.md

```diff
@@ -24,7 +24,7 @@ Please use one of these hashtags for your PR title:
   - [ ] 10.13
   - [ ] 10.14
   - [ ] 10.15
-  - [ ] 11.0
+  - [ ] 11.1
```

Comment:
Probably should just make this 11, as Apple now appears to be using all three build numbers unlike they really only used the last 2

Reply:
Could also add localization checkboxes for the testing things so we make sure things are tested in various localizations?